modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,177 @@
+= {oadp-short} use case for two BSLs
+
+In this use case, you configure the DPA with two storage locations by using two cloud credentials. You backup a `MySQL` database application by using the default BSL. {oadp-short} stores the backup resources in the default BSL. You then backup the `MySQL` application again by using the second BSL.
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must configure two backup storage locations, AWS S3 and Multicloud Object Gateway (MCG).
+* You must have a `MySQL` database application deployed on a Red Hat OpenShift Cluster.
+
+.Procedure
+
+. Create the first `Secret` for AWS S3 storage provider with the default name by running the following command.
++
+[source, terminal]
+----
+oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=<aws_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for AWS S3.
+
+. Create the second `Secret` for MCG with a custom name by running the following command.
++
+[source, terminal]
+----
+oc create secret generic mcg-secret -n openshift-adp --from-file cloud=<MCG_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for MCG. Note the name of the custom secret `mcg-secret`.
+
+. Configure the DPA with the two BSLs as shown in the following example.
++
+[source, yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: two-bsl-dpa
+  namespace: openshift-adp  
+spec:
+  backupLocations:
+  - name: aws
+    velero:
+      config:
+        profile: default
+        region: <region> # <1>
+      credential:
+        key: cloud
+        name: cloud-credentials
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: velero
+      provider: aws
+  - name: mcg
+    velero:
+      config:
+        insecureSkipTLSVerify: "true"
+        profile: noobaa
+        region: noobaa
+        s3ForcePathStyle: "true"
+        s3Url: <s3_url> # <3>
+      credential:
+        key: cloud
+        name: mcg-secret # <4>
+      objectStorage:
+        bucket: <bucket_name_mcg> # <5>
+        prefix: velero
+      provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+----
+<1> Specify the AWS region for the bucket.
+<2> Specify the AWS S3 bucket name.
+<3> Specify the URL of the S3 endpoint for MCG.
+<4> Specify the name of the custom secret for MCG storage.
+<5> Specify the MCG bucket name.
+
+. Create the DPA by running the following command.
++
+[source, terminal]
+----
+oc create -f <dpa_file_name> # <1>
+----
+<1> Specify the file name of the DPA you configured in an earlier step.
+
+. Verify if the DPA has reconciled by running the following command.
++
+[source, terminal]
+----
+oc get dpa -o yaml
+----
+
+. Verify if the BSL status is available by running the following command.
```

Comment:
Since are 2 BSLs, this should be in plural, right?

Reply:
I would also like to see some sample output of the `oc get bsl` command here.

Reply:
QE was unable to spin up a cluster with 2 BSLs today due to Jenkins issues. I am sending the PR for OCP peer review and add the output of the bsl command when the PR is back from OCP peer review.